### PR TITLE
Add Qmd to list of Common R source files

### PR DESCRIPTION
**Pull request by jgutman**
### Intent

Addresses #10469. Tweaked from "Common R source files" to "Common source files", replaced Rmd with Markdown (includes *.md, *.rmd, *.qmd), added yml/yaml and js as possible source files as well.

### CODE REVIEW

**Review by romainfrancois — APPROVED** [review]
I don't have a strong opinion on whether `*.md` should be mentioned. 

If we search for them, perhaps they should be mentioned, but they're not really "source files"

**Comment by jgutman** [comment]
Yeah exactly, but we do search for them, which I found kind of surprising. Alternatively, we could NOT search for them -- I'm assuming users haven't been relying on that, because we don't say we're searching for them, but we currently do.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.ui.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
          <rw:FormListBox ui:field="listPresetFilePatterns_"
                     styleName="{style.presetFilePatterns}">
             <g:item value=""><ui:msg key="allFiles">All Files</ui:msg></g:item>
-            <g:item value="*.r, *.R, *.rnw, *.Rnw, *.rmd, *.Rmd, *.rmarkdown, *.Rmarkdown, *.qmd, *.Qmd, *.md, *.rhtml, *.Rhtml, *.h, *.hpp, *.c, *.cpp"><ui:msg key="commonRSourceFiles">Common R source files (R, C/C++, Rnw, Rmd, Rhtml)</ui:msg></g:item>
+            <g:item value="*.r, *.R, *.rnw, *.Rnw, *.rmd, *.Rmd, *.rmarkdown, *.Rmarkdown, *.qmd, *.Qmd, *.md, *.rhtml, *.Rhtml, *.h, *.hpp, *.c, *.cpp"><ui:msg key="commonRSourceFiles">Common R source files (R, C/C++, Rnw, Rmd, Rhtml, Qmd)</ui:msg></g:item>
             <g:item value="*.r, *.R"><ui:msg key="rScripts">R Scripts</ui:msg></g:item>
```
Comment (jgutman) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.ui.xml:61]: Maybe change this Markdown

**Review by kevinushey — APPROVED** [review]
LGTM!

**Comment by jgutman** [comment]
Warrants a news item before merging -- maybe in a Find in Files subsection

**Pull request merged**